src/Kanbas/Courses/Quizzes/Preview/Question/List/index.tsx

```diff
@@ -0,0 +1,14 @@
+import RightSide from "../../../../../layout/Content/NotLeftSide/RightSide";
+
+function QuizQuestionPreviewList() {
+    return (
+        <>
+            <div></div>
+            <RightSide>
```

Comment:
This is not where RightSide is supposed to be used. 
It is supposed to be used at the same level as MiddleContent.
Please refer the home page where all 3 : LeftSide, MiddleContent, and RightSide are used together.

Reply:
That's my bad, I forgot to remove after adding to the same level as MiddleContent. Good catch!